terraform/projects/app-ecs-instances/instance-user-data.tpl

```diff
@@ -75,3 +76,46 @@ echo "ECS_CLUSTER=${cluster_name}" >> /etc/ecs/ecs.config
 yum install -y ecs-init
 start ecs
 service docker start
+
+case "$AZ" in
+        "eu-west-1a")
+            dns_name="mesh-1"
+            ;;
+
+        "eu-west-1b")
+            dns_name="mesh-2"
+            ;;
+
+        "eu-west-1c")
+            dns_name="mesh-3"
+            ;;
+
+         *)
+            echo "no Dns values found"
```

Comment:
Optional: I know it's super unlikely but I wonder if our script should fail if it ever gets to this point?

Reply:
hmmm I believe that is a matter of preference. It is an iffy one. :/

should the script fail, I do not know at this stage. these are the last commands, i believe the purpose of this message is to have something in the log file. 

Why would you want the script to fail as I am currently not sure ?